config/locales/views.youth.de.yml

```diff
@@ -68,6 +68,10 @@ de:
 
     participation_contact_data:
       managed:
+        update:
+          flash:
+            privacy_policy_not_accepted: 'Um die Anmeldung abzuschliessen, muss der Datenschutzerklärung zugestimmt werden.'
```

Comment:
Das Dokument / die Dokumente müssen nicht unbedingt als "Dateschutzerklärung" benannt sein. Im UI nennen wir das bisher nur "die Bestimmungen".

Reply:
Nevermind, in der anderen Flash-Message heisst es bereits genau so.